draft-ietf-jsonpath-base.md

```diff
@@ -1361,9 +1362,47 @@ Provisional registration? (standards tree only):
 
 # Security Considerations {#Security}
 
-This section gives security considerations, as required by {{RFC3552}}.
-
-
+Security considerations for JSONPath can stem from
+
+* attack vectors on JSONPath implementations, and
+* the way JSONPath is used in security-relevant mechanisms.
+
+## Attack vectors on JSONPath Implementations
+
+Historically, JSONPath has often been implemented by feeding parts of
+the query to an underlying programming language engine, e.g.,
+JavaScript.
+This approach is well known to lead to injection attacks and would
+require perfect input validation to prevent these attacks (see
+{{Section 12 of -json}} for similar considerations for JSON itself).
+Instead, JSONPath implementations need to implement the entire syntax
+of the query without relying on the parsers of programming language
+engines.
```

Comment:
This is related to the common security command with JSON-formatted protocol elements: Thou shalt not use eval().

But… I'm thinking of evaluating JSONpaths in JS and it's not obvious where it'd be attractive to feed fragments to eval() or equivalent. So I'm not 100% convinced this needs to be said.  Or is there an obvious example that I'm missing?